app/views/articles/_bottom_content.html.erb

```diff
@@ -1,29 +1,19 @@
 <% if articles %>
-<div class="more-articles container">
-  <% articles.each do |article| %>
-    <a href="<%= article.path %>" data-preload-image="<%= cloud_cover_url(article.main_image) %>">
-      <div class="single-other-article">
-        <div class="picture">
-          <%= cl_image_tag(article.user.profile_image_url,
-                           type: "fetch",
-                           crop: "imagga_scale",
-                           width: 100,
-                           height: 100,
-                           quality: "auto",
-                           flags: "progressive",
-                           fetch_format: "auto",
-                           loading: "lazy",
-                           sign_url: true,
-                           alt: "#{article.user.username} profile image") %>
+  <div class="more-articles container">
+    <% articles.each do |article| %>
+      <a href="<%= article.path %>" data-preload-image="<%= cloud_cover_url(article.main_image) %>">
+        <div class="single-other-article">
+          <div class="picture" data-hover-details="users/<%= article.user_id %>">
+            <img src="<%= ProfileImage.new(article.cached_user).get(90) %>" alt="<%= article.cached_user.username %> profile image"/>
+          </div>
+          <div class="content">
+            <h3><%= article.title %></h3>
+            <h4 data-hover-details="users/<%= article.user_id %>">
```

Comment:
I wonder if it's not easier to just add it to the outermost element instead of repeating it

Reply:
There's no "outer element" here because the user profile image div is a sibling to the article user and title next to it. And we don't want this hover thing happening on the title itself.

In other places the profile image and user name are within the same element.